06_puppy/sirigithub/mapStore.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"fmt"
+)
+
+type MapStore map[int]Puppy
+
+func NewMapStore() MapStore {
+	return MapStore{}
+}
+
+/** CreatePuppy takes a Puppy with a user provided ID, and creates a new Puppy if
```

Comment:
Godoc comments typically have all lines prefixed with `//` (https://blog.golang.org/godoc-documenting-go-code)

Reply:
Thanks for the pointer @juliaogris. Updated.